src/modules/esl-panel-group/core/esl-panel-group.ts

```diff
@@ -294,18 +295,23 @@ export class ESLPanelGroup extends ESLBaseElement {
     $activePanels.slice(0, overflow).forEach((el) => el.hide(params));
 
     if (max <= 0) return e.preventDefault();
+
+    this._previousHeight = this.clientHeight;
   }
 
   /** Process {@link ESLPanel} show event */
-  @listen('esl:show')
-  protected _onShow(e: CustomEvent): void {
+  @listen('esl:show esl:hide')
+  protected _onStateChanged(e: CustomEvent): void {
     const panel = e.target;
     if (!this.includesPanel(panel)) return;
+    this.toggleAttribute('has-opened', panel.open);
+
     if (this.currentMode !== 'tabs') return;
```

Comment:
Sorry, I missed something, panel in accordion won't be animated anymore?

Reply:
Nope, in accordion, it is animated as usual, but the animation is handled by ESLPanels itself.